jvm-jersey/envconfig.json

```diff
@@ -0,0 +1,52 @@
+[{
+    "version": "1.0.0",
+    "runtimeVersion" : "8", 
+    "kind": "environment", 
+    "status": "Stable", 
+    "name": "JVM Environment",
+    "shortDescription": "JVM environment based on Jersey RESTful Web Services framework",  
+    "readme" : "https://github.com/fission/environments/tree/master/jvm-jersey",
+    "examples" : "https://github.com/fission/environments/tree/master/jvm-jersey/examples",
+    "keywords": [],
+    "icon": "https://upload.wikimedia.org/wikipedia/de/e/e1/Java-Logo.svg",
+    "maintainers": [
+        {
+            "name": "life1347",
+            "link": "https://github.com/life1347"
+        },
+        {
+            "name": "soamvasani",
+            "link": "https://github.com/soamvasani"
+        },
+        {
+            "name": "vishal-biyani",
+            "link": "https://github.com/vishal-biyani"
+        }
+    ]
+},
+{
+    "version": "1.0.0", 
+    "runtimeVersion" : "11", 
+    "kind": "environment", 
+    "status": "Stable", 
+    "name": "JVM Environment",
+    "shortDescription": "JVM environment based on Jersey RESTful Web Services framework",  
+    "readme" : "https://github.com/fission/environments/tree/master/jvm-jersey",
+    "examples" : "https://github.com/fission/environments/tree/master/jvm-jersey/examples",
+    "keywords": [],
+    "icon": "",
+    "maintainers": [
```

Comment:
@sahil-lakhwani Please add yourself here and remove others - JVM-Jersey is all created by you!